indico/util/fs_test.py

```diff
@@ -33,8 +33,8 @@ def test_secure_client_filename(filename, expected):
     ('foo.txt', '', 'foo.txt'),
     ('foo.txt', 'example.zip', 'foo.txt'),
     ('../../../etc/passwd', '', 'etc_passwd'),
-    ('m\xf6p.txt', '', 'moep.txt'),
-    ('/m\xf6p.txt', '', 'moep.txt'),
+    ('m\xf6p.txt', '', 'mop.txt'),
+    ('/m\xf6p.txt', '', 'mop.txt'),
```

Comment:
This is a reduction in quality. German umlauts are usually NOT transliterated by removing the diacritics, but rather in the way the previous version here used.